### ISSUE TITLE
Add error handling

### ISSUE THREAD

**Issue by janegilring** [FBoucher/AzSubcriptionCleaner#14]
The [AzSubCleanerTimeTrigger function](https://github.com/FBoucher/AzSubscriptionCleaner/blob/master/azure-function/AzSubCleanerTimeTrigger/run.ps1) should have error handling for Get-AzResource (line 17).

I was involved in a case where Get-AzResource failed on some runs, causing some resource groups which had resources to be accidentally deleted. Management locks was implemented afterwards to prevent this from happening again, but error handling should be added regardless if the environment is using Management Locks or not.

**Comment by fboucher** [comment]
Interesting!
Thanks @janegilring interested in writing the fix?

**Comment by janegilring** [comment]
@FBoucher  Yes, I can look into it and send a Pull Request when it's ready 👍
